### ISSUE TITLE
App crashes on requestThumbnail when image doesn't exist

### ISSUE THREAD

**Issue by michael-golfi** [Sh1d0w/multi_image_picker#222]
**Bug Report**

`multi_image_picker` version: 4.5.0+2 

Is it happening on Android: No 

Is it happening on iOS: Yes, on simulator

```
 <!-- run `flutter doctor` and paste the output here: -->
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.7.8+hotfix.4, on Mac OS X 10.14.6 18G87, locale en-US)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 10.3)
[✓] iOS tools - develop for iOS devices
[✓] Android Studio (version 3.4)
[✓] Android Studio (version 3.4)
[✓] VS Code (version 1.37.1)
[✓] Connected device (1 available)

• No issues found!
```

**Describe the bug**

I'm running the same code on Android and iOS. Pictures exist on Android but not iOS and requesting thumbnails when they don't exist on iOS crashes the app. I tried catching all exceptions but it doesn't help. The crash happens [here](https://github.com/Sh1d0w/multi_image_picker/blob/master/lib/src/picker.dart#L111-L117). I tried wrapping this in a try-catch for PlatformExceptions but it still crashes.

**EDIT**: My mistake, I realized I didn't have some of the right stuff in the `Info.plist`. Is there a way to still catch these sorts of errors though?

**Comment by Sh1d0w** [comment]
Hi, can you please explain how to reproduce this? How you come to state where you request thumb for asset that does not exist?

Usually when you invoke the image picker you choose the images you want from it and it returns array of assets. Right after that you can request thumbs for them.

**Comment by michael-golfi** [comment]
Sure - this does sound like an edge case. I have an app running on the iOS simulator and on a physical Android phone. Metadata for pictures is uploaded to a backend and then each app sees the same metadata. When the simulator sees it it tries looking for the picture but can't find it. I think this was more of a permissions issue - it doesn't crash anymore. Now it just has the circular loading indicator and that spins forever since it can't find it.

**Comment by Sh1d0w** [comment]
Hi, I've released version 4.5.2 which now will throw AssetNotFoundException if you try to read images that does not exists both on Android and iOS.
